src/components/Experiment/QuickInput/QuickVideoInput.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import "./QuickVideoInput.scss";
+import Task from "../../../helpers/Task";
+import useQuickInputControl from "./useQuickInputControl";
+import useBEMNaming from "../../../common/useBEMNaming";
+import { QuickInputTabContent } from "./QuickInputTabContent";
+import { QuickInputTabTitle } from "./QuickInputTabTitle";
+import { QuickInputType } from "./quickInputType";
+
+export default function QuickVideoInput(props) {
```

Comment:
Can you add a TODO comment to `QuickInput` that we can probably refactor all of these and merge `QuickText/Image/Audio/VideoInput` into one file. I'm pretty sure there's only ~2 lines of difference in each of them.